Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -1953,15 +1939,13 @@ public ChapterInfo GetChapter(BaseItem item, int index)
 
             using (var connection = GetConnection(true))
             {
-                using (var statement = PrepareStatement(connection, "select StartPositionTicks,Name,ImagePath,ImageDateModified from " + ChaptersTableName + " where ItemId = @ItemId and ChapterIndex=@ChapterIndex"))
-                {
-                    statement.TryBind("@ItemId", item.Id);
-                    statement.TryBind("@ChapterIndex", index);
+                using var statement = PrepareStatement(connection, "select StartPositionTicks,Name,ImagePath,ImageDateModified from " + ChaptersTableName + " where ItemId = @ItemId and ChapterIndex=@ChapterIndex");
```

Comment:
I think it would be better to pull this outside the braces instead of mixing formats